pkg/sync/v12/data/ValidatingWebhookConfiguration.admissionregistration.k8s.io/aro-admission-controller.aro.openshift.io.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: aro-admission-controller.aro.openshift.io
+webhooks:
+- clientConfig:
+    caBundle: null
+    service:
+      name: aro-admission-controller
+      namespace: kube-system
+      path: /pods
+  failurePolicy: Fail
+  name: pods.aro-admission-controller.aro.openshift.io
+  namespaceSelector: {}
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - pods
+- clientConfig:
+    caBundle: null
+    service:
+      name: aro-admission-controller
+      namespace: kube-system
+      path: /daemonsets
+  failurePolicy: Fail
+  name: daemonsets.aro-admission-controller.aro.openshift.io
+  namespaceSelector: {}
+  rules:
+  - apiGroups:
+    - apps
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - daemonsets
+- clientConfig:
+    caBundle: null
+    service:
+      name: aro-admission-controller
+      namespace: kube-system
+      path: /replicasets
+  failurePolicy: Fail
+  name: replicasets.aro-admission-controller.aro.openshift.io
+  namespaceSelector: {}
+  rules:
+  - apiGroups:
+    - apps
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - replicasets
```

Comment:
can you condense these hooks per apigroup and version? e.g.
```
- clientConfig:
    caBundle: null
    service:
      name: aro-admission-controller
      namespace: kube-system
      path: /daemonsets
  failurePolicy: Fail
  name: daemonsets.aro-admission-controller.aro.openshift.io
  namespaceSelector: {}
  rules:
  - apiGroups:
    - apps
    apiVersions:
    - v1
    operations:
    - CREATE
    - UPDATE
    resources:
    - daemonsets
    - replicasets
```

Reply:
Condensed.